gulpfile.js

```diff
@@ -79,6 +82,11 @@ gulp.task('fractal:build', ['inheritance', 'svg-sprite', 'sass'], () => {
 });
 
 // Gulp tasks
+gulp.task('a11y', () => {
+  fractal.components.set('default.preview', '@preview-test');
```

Comment:
Let's rename it to `preview-accessibility` or `preview-a11y`, there is possibility that we will have more tests in the future

Reply:
changed